.azure-pipelines/scripts/measure_code_cov.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I don't think it makes sense that there are new scripts which duplicate the non-coverage test logic. Why not extend the existing scripts and add an extra environment variable to enable it?

Reply:
There are key differences between this script and the test runner script.

* This script is conditioned on the fact that sgx-lkl was built with CODE_COVERAGE=1
* We ideally want to include samples in the code coverage measurement
* For each test/sample, we run `run-hw` and `run-sw` back to back, and extract the collective coverage data from the image.

If you compare this script and `test_runner`, also `measure_one_cov` with `run_test`, there aren't many duplicated code. I prefer to keep them separate for easier maintenance.

Reply:
Thanks for explaining. I'm trying to think how this fits into CI. Given that debug/release build modes have slightly different code paths it may make sense to capture code coverage in both modes and merge it. Or we just do it for one mode if the differences are not worth measuring. Either way, it would mean adding at least one new build job with CODE_COVERAGE=1 and corresponding test/coverage job(s).
You're saying that your script runs both hw and sw. This is a bit unfortunate because it prevents running sw on non-SGX machines and in parallel. Would it be possible to run them separately, dump the coverage files and merge them in a follow-up step?

Reply:
It's hard to merge coverage data from two VMs. It requires a global vm for aggregation. We will leave the work of integrating into pipeline in another PR.

Reply:
That's actually not that hard to do, you can specify dependencies between jobs and use build artifacts to upload/download job data.